recipes/PyICU/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "PyICU" %}
+{% set version = "2.0.3" %}
+{% set sha256 = "c452d14409d93819a398a93d27d5d58d6236af690d537eb2d76c8305e8d0fa5f" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - toolchain
```

Comment:
instead of using toolchain can we switch this to

```
- {{ compiler('c') }}
- {{ compiler('cxx') }}
```
and add icu to the host section